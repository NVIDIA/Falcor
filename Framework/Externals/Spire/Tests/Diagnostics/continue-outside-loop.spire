// `continue` where it isn't allowed

void foo() { continue; }
