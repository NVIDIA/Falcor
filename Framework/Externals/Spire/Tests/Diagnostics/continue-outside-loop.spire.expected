result code = -1
standard error = {
Tests/Diagnostics/continue-outside-loop.spire(3): error 30004: 'continue' must appear inside loop constructs.
}
standard output = {
}
